subsys/bluetooth/audio/cap_acceptor.c

```diff
@@ -27,6 +27,16 @@ int bt_cap_acceptor_register(const struct bt_csip_set_member_register_param *par
 	static struct bt_gatt_service cas;
 	int err;
 
+	CHECKIF(param->set_size == 0U) {
+		LOG_DBG("param->set_size shall be non-zero");
```

Comment:
should you log the value though?

Reply:
Hmm, we could, but this check is just for `0` for the value we could log here would always be `0`.

Reply:
oh, sorry, that was a silly comment :-)